src/pudl/workspace/datastore.py

```diff
@@ -534,15 +534,15 @@ def _get_pudl_in(args: dict) -> Path:
     if args.pudl_in:
         return Path(args.pudl_in)
     else:
-        return Path(pudl.workspace.setup.get_defaults()["pudl_in"])
+        return Path(pudl.workspace.setup.get_defaults()["PUDL_INPUT"])
```

Comment:
The fact that `pudl_in` and `PUDL_INPUT` both exist in the settings dict and have different semantics seems like a dangerous thing. Additionally, it seems that PUDL_INPUT is set into `data_dir`. Should we then use `data_dir` here instead of `PUDL_INPUIT`?

Reply:
It's definitely not ideal!  I went with `PUDL_INPUT` rather than `data_dir` because I think we're eventually just going to depend on a real settings class & have it take cues from the environment variables.

There's a ton of vestigial mess in this settings system and I would ❤️❤️❤️ to blow it all away and replace it with a rational Pydantic settings class, but that'll mean chasing down and removing all the `pudl_settings` dictionaries that are floating around right now which will be a bit of work (but totally worth it in the medium term...). I'm inclined toward that option rather than trying to rationalize the current mess in-place. But we haven't been able to prioritize it. 😭